mlpp_lib/datasets.py

```diff
@@ -153,22 +153,25 @@ def apply_filter(self):
         LOGGER.info("Applying filter to features and targets.")
         self.x, self.y = self.filter.apply(self.x, self.y)
 
-    def standardize(self, stage=None):
-        LOGGER.info("Standardizing data.")
-        if self.standardizer is None:
-            if stage == "test":
-                raise ValueError("Must provide standardizer for `test` stage.")
+    def normalize(self, stage=None):
+        LOGGER.info("Normalizing data.")
 
-            self.standardizer = Standardizer()
-            self.standardizer.fit(self.train[0])
+        if self.normalizer is None:
+            if stage == "test":
+                raise ValueError("Must provide normalizer for `test` stage.")
+            else:
+                self.normalizer = DataTransformer(
+                    {"Identity": (list(self.train[0].data_vars), {})}
```

Comment:
Using standard transformation as default keeps current behaviour unchanged
```suggestion
                    {"Standardizer": (list(self.train[0].data_vars), {})}
```